qa/L0_simple_cuda_shared_memory_example/test.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
Isn't this file already in the simple CUDA shared memory client PR?

Reply:
Yes, removed from this PR